### ISSUE TITLE
Translation.byLocale

### ISSUE THREAD

**Issue by StevenAtAskAttis** [marcglasberg/i18n_extension#49]
Hi,

I'm trying the Translation by locale:

```
extension Localization on String {
  static var _t = Translations.byLocale("en") +
      {
        "en": {
          "Hi": "Hi",
          "Goodbye.": "Goodbye.",
        },
      } +
      {
        "nl": {
          "Hi": "Hallo",
          "Goodbye.": "Vaarwel",
        }
      };

  String get i18n => localize(this, _t);
}
```
and implement
```
return I18n(
      initialLocale: Locale("nl"),
      ...
                          Text(
                            'Hi'.i18n,
                          ),
     ...
```
but i get the following error

```
The following NoSuchMethodError was thrown building Consumer<RegisterViewModel>(dirty, dependencies: [_InheritedProviderScope<RegisterViewModel>]):
The method '[]' was called on null.
Receiver: null
Tried calling: []("Hi")
```

It does work when I use to normal way
```
import 'package:i18n_extension/i18n_extension.dart';

extension Localization on String {
  static var _t = Translations("en") +
      {
        "en": "Hi",
        "nl": "Hallo",
      };

  String get i18n => localize(this, _t);
}
```

What am I doing wrong?
I would want to use the byLocale because I think it's easier to maintain.

Thanks for any help
regards
Steven

**Comment by marcglasberg** [comment]
Hi Steven, could you please provide me with some minimal code (the shortest possible quantity of code), runnable (with a main method and all), in a single file, that demonstrates the problem you are having?

Please post the code here.

**Comment by StevenAtAskAttis** [comment]
Pffff,
I really don't know what happend. I started with reproducing the issue with minimum code and off coarse it worked. So I couldn't reproduce it.

Then I went back to my code and first it gave the same error again, and then suddenly, don't ask why, bam everything works even with plural.

So, I really don't know what went wrong.
probably me being a NOOB, but it's weird.

Anyway thanks, because by asking your question I could 'solve' it :-P

regards
Steven
